charts/pega/templates/_helpers.tpl

```diff
@@ -48,6 +48,22 @@ false
 {{- end }}
 {{- end }}
 
+{{- define "pegaTomcatKeystoreSecret" }}
+{{- $depName := printf "%s" (include "deploymentName" $) -}}
+{{- $depName -}}-tomcat-keystore-secret
+{{- end }}
+
+{{- define "pegaVolumeTomcatKeystore" }}pega-volume-tomcat-keystore{{- end }}
+
+{{- define "pegaTomcatKeystoreTemplate" }}
```

Comment:
rename to `pegaVolumeTomcatKeystoreTemplate`? if its only referring to volume.

Reply:
done